schedule-replicas-by-topology-labels.md

```diff
@@ -0,0 +1,135 @@
+
+Assume that the topology has three layers: zone > rack > host, and you can use these labels (zone, rack, host) to set the TiKV location in one of the following methods:
+
++ Use the command-line flag:
+
+    {{< copyable "" >}}
+
+    ```
+    tikv-server --labels zone=<zone>,rack=<rack>,host=<host>
+    ```
+
++ Configure in the TiKV configuration file:
+
+    {{< copyable "" >}}
+
+    ```toml
+    [server]
+    labels = "zone=<zone>,rack=<rack>,host=<host>"
+    ```
+
+### Configure `location-labels` for PD
+
+According to the description above, the label can be any key-value pair used to describe TiKV attributes. But PD cannot identify the location-related labels and the layer relationship of these labels. Therefore, you need to make the following configuration for PD to understand the TiKV node topology.
+
++ If the PD cluster is not initialized, configure `location-labels` in the PD configuration file:
+
+    {{< copyable "" >}}
+
+    ```toml
+    [replication]
+    location-labels = ["zone", "rack", "host"]
+    ```
+
++ If the PD cluster is already initialized, use the pd-ctl tool to make online changes:
+
+    {{< copyable "shell-regular" >}}
+
+    ```bash
+    pd-ctl config set location-labels zone,rack,host
+    ```
+
+The `location-labels` configuration is an array of strings, and each item corresponds to the key of TiKV `labels`. The sequence of each key represents the layer relationship of different labels.
+
+> **Note:**
+>
+> You must configure `location-labels` for PD and `labels` for TiKV at the same time for the configurations to take effect. Otherwise, PD does not perform scheduling according to the topology.
+
+### Configure `isolation-level` for PD
+
+If `location-labels` has been configured, you can further enhance the topological isolation requirements on TiKV clusters by configuring `isolation-level` in the PD configuration file.
+
+Assume that you have made a three-layer cluster topology by configuring `location-labels` according to the instructions above: zone -> rack -> host, you can configure the `isolation-level` to `zone` as follows:
+
+{{< copyable "" >}}
+
+```toml
+[replication]
+isolation-level = "zone"
+```
+
+If the PD cluster is already initialized, you need to use the pd-ctl tool to make online changes:
+
+{{< copyable "shell-regular" >}}
+
+```bash
+pd-ctl config set isolation-level zone
+```
+
+The `location-level` configuration is an array of strings, which needs to correspond to a key of `location-labels`. This parameter limits the minimum and mandatory isolation level requirements on TiKV topology clusters.
+
+> **Note:**
+>
+> `isolation-level` is empty by default, which means there is no mandatory restriction on the isolation level. To set it, you need to configure `location-labels` for PD and ensure that the value of `isolation-level` is one of `location-labels` names.
+
```

Comment:
@ireneontheway Please align https://github.com/pingcap/docs-cn/pull/4502 in this document, thanks!

Reply:
Done in https://github.com/pingcap/docs/pull/3825/commits/dbd532133b09dbae13883a698ef08a13a73906ff